sgds/sass/sgds-theme/components/_accordion.scss

```diff
@@ -0,0 +1,17 @@
+.sgds{
+    &.accordion{
+        line-height: 2rem;
```

Comment:
Should we remove this? Seems like this doesn't affect any changes to the styling.

Reply:
its changing the accordion-body line-height